certified-connectors/RamQuest Actions - UAT/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1557 @@
+            "$ref": "#/definitions/PubCS.TitleOrder.Api.Models.TitleOrderCommitmentSchedA"
+          }
+        },
+        "scheduleB": {
+          "description": "Gets or Sets ScheduleB",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/PubCS.TitleOrder.Api.Models.TitleOrderCommitmentSchedB"
+          }
+        },
+        "scheduleCD": {
+          "description": "Gets or Sets ScheduleCD",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/PubCS.TitleOrder.Api.Models.TitleOrderCommitmentSchedCD"
+          }
+        }
+      }
+    },
+    "PubCS.TitleOrder.Api.Models.TitleOrderOwnerPolicyDetail": {
+      "description": "",
+      "type": "object",
+      "properties": {
+        "titleVestedIn": {
+          "description": "Gets or Sets TitleVestedIn",
+          "type": "string"
+        },
+        "policyNumber": {
+          "description": "Gets or Sets PolicyNumber",
+          "type": "string"
+        },
+        "dates": {
+          "description": "Gets or Sets Dates",
+          "type": "array",
+          "items": {
+            "format": "date-time",
+            "type": "string"
+          }
+        },
+        "strikes": {
+          "description": "Gets or Sets Strikes",
+          "type": "object",
+          "additionalProperties": {
+            "type": "boolean"
+          }
+        },
+        "assignedLanguage": {
+          "description": "Gets or Sets AssignedLanguage",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/PubCS.TitleOrder.Api.Models.StandardLanguage"
+          }
+        }
+      }
+    },
+    "PubCS.TitleOrder.Api.Models.TitleOrderLoanPolicyDetail": {
+      "description": "",
+      "type": "object",
+      "properties": {
+        "sequence": {
+          "format": "int32",
+          "description": "Gets or Sets Sequence",
+          "type": "integer"
+        },
+        "loanSequenceId": {
+          "format": "int32",
+          "description": "Gets or Sets LoanSequenceId",
+          "type": "integer"
+        },
+        "policyNumber": {
+          "description": "Gets or Sets PolicyNumber",
+          "type": "string"
+        },
+        "dates": {
+          "description": "Gets or Sets Dates",
+          "type": "array",
+          "items": {
+            "format": "date-time",
+            "type": "string"
+          }
+        },
+        "strikes": {
+          "description": "Gets or Sets Strikes",
+          "type": "object",
+          "additionalProperties": {
+            "type": "boolean"
+          }
+        },
+        "assignedLanguage": {
+          "description": "Gets or Sets AssignedLanguage",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/PubCS.TitleOrder.Api.Models.StandardLanguage"
+          }
+        }
+      }
+    },
+    "PubCS.TitleOrder.Api.Models.TitleOrderPolicy": {
+      "description": "",
```

Comment:
please use a proper description for the response. you can use something like "OK", "Sucess", "Bad request", etc.